substrafl_examples/get_started/run_mnist_torch.py

```diff
@@ -190,32 +191,15 @@ def accuracy(datasamples, predictions_path):
     return accuracy_score(y_true, np.argmax(y_pred, axis=1))
 
 
-# %%
-# We also need to specify the third parties dependencies required to compute the metrics.
-# The :ref:`substrafl_doc/api/dependency:Dependency` object is instantiated in order to install the right libraries in
-# the Python environment of each organization.
-#
-# As for the dataset, we also define :ref:`documentation/concepts:Permissions`.
-
-from substrafl.dependency import Dependency
-
-metric_deps = Dependency(pypi_dependencies=["numpy==1.23.1", "scikit-learn==1.1.1"])
+def roc_auc(datasamples, predictions_path):
+    y_true = datasamples["labels"]
 
-permissions_metric = Permissions(public=False, authorized_ids=[ALGO_ORG_ID] + DATA_PROVIDER_ORGS_ID)
+    n_class = np.max(y_true) + 1
+    y_true_one_hot = np.eye(n_class)[y_true]
 
-# %%
-# After defining the metrics, dependencies, and permissions, we use the ``add_metric`` function to register the metric.
-# This metric will be used on the test datasamples to evaluate the model performances.
+    y_pred = np.load(predictions_path)
```

Comment:
Minor and subjective, but for symmetry reasons with the `accuracy` definition, I would start with 
```
y_true = datasamples["labels"]
y_pred = np.load(predictions_path)
```